cmd/drone-server/inject_login.go

```diff
@@ -164,13 +164,21 @@ func provideStashLogin(config config.Config) login.Middleware {
 // provideRefresher is a Wire provider function that returns
 // an oauth token refresher for Bitbucket.
```

Comment:
Change comment to `... refresher for Bitbucket and Gitea` ?

Reply:
updated. Thanks :)